### PR TITLE
[Finishes #103347498] Enable the "Version History" button on App Lab standalone projects

### PR DESCRIPTION
**Pull request by joshlory**
Standalone App Lab projects already have versions, but we don't expose the "Version History" button.  This change replaces "Clear Puzzle" with "Version History" on those pages.

### CODE REVIEW

**Comment by davidsbailey** [comment]
Cool! It looks like you will have to update this test:

```
features/applab/scenarios.feature:56:    And I press "clear-puzzle-header"
```

**Comment by davidsbailey** [comment]
Heads up @mehalshah and @philbogle (Devs of the day) that this change is expected to affect applab eyes tests.

File: shared/js/initApp.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ window.apps = {
         if (window.dashboard.isChrome34) {
           chrome34Fix.fixup();
         }
-        if (appOptions.level.projectTemplateLevelName) {
+        if (appOptions.level.projectTemplateLevelName || appOptions.app === 'applab') {
```
Comment (davidsbailey) [inline shared/js/initApp.js:36]: Note to future selves: when we decide to enable this for Pixelation, we can just change this to `appOptions.isChannelBacked`

**Comment by davidsbailey** [comment]
LGTM after fixing possible UI test break.

**Comment by sfilman** [comment]
@joshlory would be great to have this one in for the bug bash tomorrow (at least available in levelbuilder). Possible to get this in today?

**Comment by joshlory** [comment]
@sfilman It's unlikely I'll be able to ship this today.  I still need to update the UI tests, and I also have dev-of-day work on my plate first.  This isn't a change we want to rush in before the staging lockdown tomorrow.

**Comment by sfilman** [comment]
Thanks for the update, Josh. Makes sense. We can catch on the next bug bash.

**Comment by sfilman** [comment]
@joshlory will you be able to wrap this one this week?

**Comment by joshlory** [comment]
Yep, fixing up UI tests now.

**Pull request merged**